R/generics.R

```diff
@@ -8,8 +8,8 @@ NULL
 #' @rdname as.CellGraphAssay5
 #' @export as.CellGraphAssay5
 as.CellGraphAssay5 <- function (
-  x,
-  ...
+    x,
```

Comment:
Our current style is to use an indentation of 2 for function arguments. No need to change these to 4.

Reply:
Fixed!